examples/product-documents/ecs/usage-examples/python/demo/deploy_svn_by_using_svnserve_on_ECS.py

```diff
@@ -0,0 +1,167 @@
+                                               )
+
+        # ECS image ID parameter
+        image_id_param = core.RosParameter(self,
+                                           "ImageId",
+                                           default_value='centos_7_04_64_20G_alibase_201701015.vhd',
+                                           type=core.RosParameterType.STRING,
+                                           description={
+                                               "zh-cn": "镜像ID",
+                                               "en": "Image ID"
+                                           },
+                                           label="公共镜像CentOS_7_4_64位"
+                                           )
+        # ECS instance type parameter
+        instance_type_param = core.RosParameter(self,
+                                                "InstanceType",
+                                                association_property='ALIYUN::ECS::Instance::InstanceType',
+                                                association_property_metadata={
+                                                    "ZoneId": "ZoneId"
+                                                },
+                                                default_value="ecs.c6.large",
+                                                label="Instance Type",
+                                                type=core.RosParameterType.STRING,
+                                                description={
+                                                    "zh-cn": "<font color='blue'><b>1.选择机型前请先确认当前可用区下该机型是否有货，部分机型需要提前报备</b></font><br><font color='blue'><b>2.可选机型列表</font><br></b></font>[ecs.c5.large <font color='green'>2vCPU 4GiB 内网带宽1Gbps 内网收发包30万PPS</font>]<br></b>[ecs.c5.xlarge <font color='green'>4vCPU 8GiB 内网带宽1.5Gbps 内网收发包50万PPS</font>]<br></b>[ecs.c5.2xlarge <font color='green'>8vCPU 16GiB 内网带宽2.5Gbps 内网收发包80万PPS</font>]",
+                                                    "en": "<font color='blue'><b>1.Before selecting the model please confirm that the current available zone under the model is in stock, some models need to be reported in advance</b></font><br><font color='blue'><b>2.List of optional models</font><br></b></font>[ecs.c5.large <font color='green'>2vCPU 4GiB Intranet bandwidth1Gbps In-grid sending and receiving packages30MillionPPS</font>]<br></b>[ecs.c5.xlarge <font color='green'>4vCPU 8GiB Intranet bandwidth1.5Gbps In-grid sending and receiving packages50MillionPPS</font>]<br></b>[ecs.c5.2xlarge <font color='green'>8vCPU 16GiB Intranet bandwidth2.5Gbps In-grid sending and receiving packages80MillionPPS</font>]"
+                                                })
+
+        # Zone id parameter
+        zone_id_param = core.RosParameter(
+            self,
+            "ZoneId",
+            type=core.RosParameterType.STRING,
+            association_property="ALIYUN::ECS::Instance::ZoneId",
+            default_value="cn-hangzhou-k"
```

Comment:
Please do not set a default value for ZoneId, as this will not work for users switching between regions.

Reply:
Removed the default value for zone id